MathProblems/HammingDistance.py

```
@@ -0,0 +1,22 @@
+                   x=[0]+x 
+             
+          elif len(y)<len(x):
+               while len(y)<len(x):
+                   y=['0']+y 
+          count=0
+          index=0 
+          while index<len(y):
+                if x[index]!=y[index]:
+                    count+=1
+                index+=1
+          print(x)
```

Flake8 (7.2.0)

[error] indentation is not a multiple of 4

(E111)